entries/error.xml

```diff
@@ -52,6 +52,15 @@ $( "img" )
   })
   .attr( "src", "missing.png" );
 ]]></code>
+    <p>In the example, missing.png is an hypothetical missing image that is used to trigger the <code>error</code> event, not an image to replace those giving an error. To do that, you have to modify the body of the anonymous function passed to <code>.error()</code> as shown below:</p>
+    <code><![CDATA[
+$( "img" )
+  .error(function() {
+    $( this ).attr( "src", "replacement.png" );
+  })
+  .attr( "src", "missing.png" );
+]]></code>
+  <p><b>Note:</b> Be sure that the replacement image exists; otherwise the <code>error</code> event will be triggered indefinitely.</p>
```

Comment:
Also, I don't think `<p>` tags are allowed in `<example>`s. This just results into this: http://gyazo.com/f7795ca0fd998b6701b68bbb322fed0d (notice your extra info inside the `<p>` isn't rendered).

@AurelioDeRosa Do you have a working local docs setup? Normally you should be able to repro that there. If you don't, ping me on IRC and I'll  be happy to help you out.